### PR TITLE
Projectiles

### PR DESCRIPTION
**Pull request by n0remac**

### CODE REVIEW

File: Core/Projectiles/Projectile_Manager.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        # to add in self.view_bottom and self.view_left.
+        dest_x = x
+        dest_y = y
+
+        # Do math to calculate how to get the bullet to the destination.
+        # Calculation the angle in radians between the start points
+        # and end points. This is the angle the bullet will travel.
+        x_diff = dest_x - start_x
+        y_diff = dest_y - start_y
+        angle = math.atan2(y_diff, x_diff)
```
Comment (freeqaz) [inline Core/Projectiles/Projectile_Manager.py:58]: Nice maths here. Did you figure this out yourself or were you able to use a guide? Just curious!

File: Core/Projectiles/Projectile_Manager.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            collision_type="wall",
+            body_type=arcade.PymunkPhysicsEngine.STATIC,
+        )
+
+        def wall_hit_handler(bullet_sprite, _wall_sprite, _arbiter, _space, _data):
+            """ Called for bullet/wall collision """
+            bullet_sprite.remove_from_sprite_lists()
+
+        self.projectile_physics.add_collision_handler(
+            "bullet", "wall", post_handler=wall_hit_handler
```
Comment (freeqaz) [inline Core/Projectiles/Projectile_Manager.py:24]: This is really simple. I love it

**Comment by freeqaz** [comment]
I tested this out and it's pretty slick. Very nice!

**Review by freeqaz — APPROVED** [review]

**Comment by n0remac** [comment]
All that was out of the tutorial. It's basic arcade stuff.

**Pull request merged**